packages/common/core-utils/src/test/events/events.spec.ts

```diff
@@ -5,7 +5,7 @@
 
 import { strict as assert } from "node:assert";
 
-import { validateAssertionError } from "@fluidframework/test-runtime-utils/internal";
+// import { validateAssertionError } from "@fluidframework/test-runtime-utils/internal";
```

Comment:
`validateAssertionError` has been replaced with the standard `Error` type because test-runtime-utils cannot be added as a dependency in core-utils, due to a layer-check failure.

fyi, @noencke